deploy/crds/app.kiegroup.org_kogitoapps_crd.yaml

```diff
@@ -157,21 +221,120 @@ spec:
+                            description: Version of the schema the FieldPath is written
+                              in terms of, defaults to "v1".
+                            type: string
+                          fieldPath:
+                            description: Path of the field to select in the specified
+                              API version.
+                            type: string
+                        required:
+                        - fieldPath
+                        type: object
+                      resourceFieldRef:
+                        description: 'Selects a resource of the container: only resources
+                          limits and requests (limits.cpu, limits.memory, limits.ephemeral-storage,
+                          requests.cpu, requests.memory and requests.ephemeral-storage)
+                          are currently supported.'
+                        properties:
+                          containerName:
+                            description: 'Container name: required for volumes, optional
+                              for env vars'
+                            type: string
+                          divisor:
+                            description: Specifies the output format of the exposed
+                              resources, defaults to "1"
+                            type: string
+                          resource:
+                            description: 'Required: resource to select'
+                            type: string
+                        required:
+                        - resource
+                        type: object
+                      secretKeyRef:
+                        description: Selects a key of a secret in the pod's namespace
+                        properties:
+                          key:
+                            description: The key of the secret to select from.  Must
+                              be a valid secret key.
+                            type: string
+                          name:
+                            description: 'Name of the referent. More info: https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#names
+                              TODO: Add other useful fields. apiVersion, kind, uid?'
+                            type: string
+                          optional:
+                            description: Specify whether the Secret or its key must
+                              be defined
+                            type: boolean
+                        required:
+                        - key
+                        type: object
+                    type: object
+                required:
+                - name
                 type: object
               type: array
+            image:
+              description: 'Image definition for the service. Example: Domain: quay.io,
+                Namespace: kiegroup, Name: kogito-jobs-service, Tag: latest. On OpenShift
```

Comment:
`Name: kogito-jobs-service`
copy / paste ? :)

Reply:
It's a description example. This file is auto generated based on the domain model. Since now we're using the same domain model for every CR, we have the same description. We might need to change this to a more "generic" example, though.

Reply:
Changed it to [generic example](https://github.com/kiegroup/kogito-cloud-operator/blob/6751a35bf6799b50747e9e5566c06cfe2ea8943f/pkg/apis/app/v1alpha1/kogitoservices_types.go#L123).